cli/src/tests/query_test.rs

```diff
@@ -2843,7 +2843,7 @@ fn test_query_captures_with_text_conditions() {
              (#eq? @function.builtin "require"))
 
             ((identifier) @variable.builtin
-              (#any-of? @variable.builtin
+              (#eq? @variable.builtin
                         "arguments"
                         "module"
                         "console"
```

Comment:
This doesn't look clear, when I look on `(#eq? @node "foo" "bar" "baz")` it isn't clear what to expect, would be match over any of values, or it expects three nodes much to all values.

Reply:
Well, the latter doesn't really make sense, so the former is the only meaningful (and hopefully documented) Interpretation...

Reply:
> the latter doesn't really make sense

why not? If it provides ordering and I for ex. wants to find all funcs with such parameter names in a row.

In any way I think it's not a good idea to add an additional sense on simple and clear things, the `#eq?` in it current form doesn't have sense when a pattern match captures more then a one capture by the same `@name` and such predicate should just filter out the whole pattern, to match in a more fine grained way there are other predicates.

I think predicate names should not be overloaded with many senses, it's better to have more precise predicates.